requirements/tools.in

```diff
@@ -11,7 +11,7 @@ flake8-bugbear
 flake8-comprehensions
 flake8-docstrings
 flake8-mutable
-ipython
+ipython==7.16.1  # 7.17 drops support for Python 3.6
```

Comment:
```suggestion
ipython <7.17  # 7.17 drops support for Python 3.6
```

to allow for patch releases?

Reply:
As far as I know IPython only patches the latest minor version (except for explicit LTS versions, which I think is only 5.x as the last py2-compatible version), but it's a good suggestion!  Done 🙂